pkg/restore/actions/pod_volume_restore_action.go

```diff
@@ -119,6 +119,28 @@ func (a *PodVolumeRestoreAction) Execute(input *velero.RestoreItemActionExecuteI
 
 	log.Info("Pod volume backups for pod found")
 
+	// Check if any of the volumes need file system restores
+	// If the restore is using native datamover or CSI, we don't need the init container
+	needsFileSystemRestore := false
```

Comment:
It seems we need to make some changes here.
To me, the issue was that the pod previously restored by fs-backup and backed up again with other ways should not be restored with the restore-wait InitContainer.

There are two ways to fix this.
* First, we can remove the unnecessary InitContainer during backup.
* Second, we can delete the InitContainer in this pod restore action.

Reply:
Thanks for feedback